examples/any-data-type/any_data_type.description

```diff
@@ -0,0 +1,5 @@
+// The `anydata` type is a sub set of type `any` of the Ballerina data types. `anydata` is an union type which consists
+// boolean | int | float | decimal | string | xml | table | anydata[] | map<anydata> types.
```

Comment:
`nil` should be here as well. 
```suggestion
// `boolean | int | float | decimal | string | xml | table | anydata[] | map<anydata> | ()` types.
```
Also shall we rephrase this to "The `anydata` type is defined by the following union: `boolean | int | float | decimal | string | xml | table | anydata[] | map<anydata> | ()`"

Reply:
done